modules/ROOT/pages/merge-types.adoc

```diff
@@ -70,9 +68,12 @@ Additionally, because both types also have a default query method that retrieves
 
 The result is a single, merged, enriched type in the unified schema.
 
-Whichever type is added first to the unified schema has the advantage of defining the name for the merged type and the corresponding default query method. In this example, because the Customer API was added first, the resulting type in the unified schema is called `Customer`, containing all the fields from both Customer and UserAccount types, and the resulting query method is called `customersById`, returning the merged `Customer` type:
+=== How Types are Ordered When Merged
+
+Whichever type is added first to the unified schema has the advantage of defining the name for the merged type and the corresponding default query method. 
+
+In this example, because the Customer API was added first, the resulting type in the unified schema is called `Customer`, containing all the fields from both Customer and UserAccount types, and the resulting query method is called `customersById`, returning the merged `Customer` type:
```

Comment:
As you write, take a look at heavily modified sentences and consider breaking them up:
```suggestion
In this example, because the Customer API type was added first, the resulting type in the unified schema is called `Customer`, containing all the fields from both Customer and UserAccount types. The resulting query method is called `customersById`, returning the merged `Customer` type:
```